asv_bench/benchmarks/series_methods.py

```diff
@@ -36,6 +36,19 @@ def time_isin(self, dtypes):
         self.s.isin(self.values)
 
 
+class IsInBoolean:
+
+    params = ["boolean", "bool"]
```

Comment:
add Boolean

Reply:
```Python
Series(np.random.randint(0, 2, 100000)).astype("Boolean")
```
This will cause an error:
```
TypeError: data type 'Boolean' not understood
```

Reply:
Yes, this is fine ("Boolean" does indeed not exist)